xarray/core/rolling.py

```diff
@@ -50,6 +50,80 @@ def __init__(self, obj, min_periods=None, center=False, **windows):
         Returns
         -------
         rolling : type of input argument
+        
+        Examples
+        -------
+        Create rolling seasonal average of monthly data e.g. DJF, JFM, ..., SON
+        
+        >>> lat = np.linspace(-90, 90, num=181)
+        >>> lon = np.linspace(0, 359, num=360)
+        >>> time = pd.date_range('15/12/1999', 
+                                 periods=12, freq=pd.DateOffset(months=1))
+        >>> a = np.repeat(np.repeat(np.linspace(0,11,num=12)[:, np.newaxis], 
+                          len(lat), axis=1)[:, :, np.newaxis], len(lon), axis=2)
+        >>> da = xr.DataArray(a, coords=[time, lat, lon], dims=['time','lat','lon'])
+        >>> da
```

Comment:
This is a fine example, but it takes a lot of space to show it because of the unused lat/lon dimensions. Maybe it would be better to drop those dimensions and make the array only go along time?